app/src/main/res/layout/list_footer.xml

```diff
@@ -27,7 +27,9 @@
 			 android:layout_weight="1" 
 			 />
 		
-		<include layout="@layout/footer" 
+		<include layout="@layout/footer"
+			android:layout_width="match_parent"
+			android:layout_height="wrap_content"
```

Comment:
These values are included in `footer.xml`: https://github.com/konklone/congress-android/blob/master/app/src/main/res/layout/footer.xml

Does the linter still want to see these expressed in the parent layout that includes it?

Reply:
Android Lint asked for it; I'm not sure if that's truly required, or if the linter isn't doing a sufficiently deep analysis.